spec/controllers/admin/domain_blocks_controller_spec.rb

```diff
@@ -8,17 +8,23 @@
   end
 
   describe 'GET #index' do
-    it 'returns http success' do
-      get :index
+    it 'renders domain blocks' do
+      42.times { Fabricate(:domain_block) }
```

Comment:
Instead of creating so many records, can you modify the limit value constant to force a page 2?